tests/tcl/tests/unit/type/hash.tcl

```diff
@@ -335,8 +335,8 @@ start_server {tags {"hash"}} {
         catch {r hincrby smallhash str 1} smallerr
         catch {r hincrby smallhash str 1} bigerr
         set rv {}
-        lappend rv [string match "ERR*not an integer*" $smallerr]
-        lappend rv [string match "ERR*not an integer*" $bigerr]
+        lappend rv [string match "ERR*non-integer*" $smallerr]
+        lappend rv [string match "ERR*non-integer*" $bigerr]
```

Comment:
why do you need these changes?

Reply:
> why do you need these changes?

Use the `ToStatus()` method of class `StatusOr` to get error information. It is expressed slightly differently than it was hardcoded before.

Reply:
I think these changes are acceptable. These messages come from `ParseInt`, and are forwarded to these callers.